docker-compose-ee.yml

```diff
@@ -0,0 +1,36 @@
+services:
+  ghsci:
+    image: ryntrnr/global-indicators-earth-engine:v4.5.2
```

Comment:
I get this is where the image currently, probably fine to keep, but idea is we will replace this with our own image.  I wonder about version being 4.5.2 -- makes it sound like its not up to date with the current docker image ([4.7.2](https://hub.docker.com/repository/docker/globalhealthyliveablecities/global-indicators/general) at time of writing).  Best to make sure the Dockerfile is based on the most recent version and environment.yml to ensure equivalence